Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -84,7 +84,7 @@ def run
             test-bot:
               strategy:
                 matrix:
-                  os: [ubuntu-22.04, macos-13, macos-14]
+                  os: [ubuntu-22.04, macos-13, macos-15]
```

Comment:
Why not:
```suggestion
                  os: [ubuntu-22.04, macos-14, macos-15]
```

Reply:
macOS 13 is x86_64. macOS 14 and 15 are arm64.

Reply:
Because `macos-13` runners are the last GitHub-hosted x86_64 runners. 14 and 15 are on arm64.

Reply:
(Such a n00b, right there in the PR description all along… 🤦‍♂️ )